experiments/generate_mutants.py

```diff
@@ -151,7 +153,12 @@ def main():
                 diff = str(client_boggart.mutations_to_diff(snapshot, [mutation]))
                 acceptable.append(diff)
 
-        yml = [{'diff': PreservedScalarString(d)} for d in acceptable]
+        yml = []
+        for diff in acceptable:
+            entry = {'diff': PreservedScalarString(diff),
+                     'uid': mutation_to_uid(diff)}
```

Comment:
@ChrisTimperley  just to make sure, isn't `mutation_to_uid(diff)` going to be different than `mutation_to_uid(PSS(diff))`? if so our scripts are going to be inconsistent.